### ISSUE TITLE
Incorrect Class PURLs have bco.owl embedded

### ISSUE THREAD

**Issue by cmungall** [BiodiversityOntologies/bco#94]
Working with @kaiiam we noticed there are some class PURLs that don't follow standard OBO syntax, I assume this is unintentional:

```
$ grep bco.owl/ target/triffo/src/ontology/mirror/bco.owl 
        <obo:IAO_0000231 rdf:resource="http://purl.obolibrary.org/obo/bco.owl/BCO_0000075"/>
        <obo:IAO_0000231 rdf:resource="http://purl.obolibrary.org/obo/bco.owl/BCO_0000075"/>
        <obo:IAO_0000231 rdf:resource="http://purl.obolibrary.org/obo/bco.owl/BCO_0000075"/>
        <obo:IAO_0000231 rdf:resource="http://purl.obolibrary.org/obo/bco.owl/BCO_0000075"/>
        <obo:IAO_0000231 rdf:resource="http://purl.obolibrary.org/obo/bco.owl/BCO_0000075"/>
        <obo:IAO_0000231 rdf:resource="http://purl.obolibrary.org/obo/bco.owl/BCO_0000075"/>
        <obo:IAO_0000231 rdf:resource="http://purl.obolibrary.org/obo/bco.owl/BCO_0000075"/>
        <obo:IAO_0000231 rdf:resource="http://purl.obolibrary.org/obo/bco.owl/BCO_0000075"/>
    <!-- http://purl.obolibrary.org/obo/bco.owl/BCO_0000080 -->
    <owl:Class rdf:about="http://purl.obolibrary.org/obo/bco.owl/BCO_0000080">
    <!-- http://purl.obolibrary.org/obo/bco.owl/BCO_0000081 -->
    <owl:Class rdf:about="http://purl.obolibrary.org/obo/bco.owl/BCO_0000081">
    <!-- http://purl.obolibrary.org/obo/bco.owl/BCO_0000075 -->
    <owl:NamedIndividual rdf:about="http://purl.obolibrary.org/obo/bco.owl/BCO_0000075">
```

**Comment by ramonawalls** [comment]
There were a handful of terms that had the incorrect purl format. I thought they had all been deprecated long ago, but it looks like I missed some. Thanks. Will fix.

**Comment by ramonawalls** [comment]
Ironic that these are new terms that were added when I was fixing the ID format. 

@robgur Heads up that two of the terms are `archeological specimen` and `zooarcheological specimen`. I will have to replace those terms. The definitions and everything will stay the same, but the IDs will change from `http://purl.obolibrary.org/obo/bco.owl/BCO_0000080` and `http://purl.obolibrary.org/obo/bco.owl/BCO_0000081` 

to 

`http://purl.obolibrary.org/obo/BCO_0000080` and `http://purl.obolibrary.org/obo/BCO_0000081`.

**Comment by ramonawalls** [comment]
Fixed during commit fbacff71208ca0d65f689e47a7417f31f33fa0ff. For some reason, I called this issue 22 in the commit message.
